SalesforceDesignSystem/Generated/SLDSColorAlias.m

```diff
@@ -24,14 +24,20 @@ + (UIColor *)sldsColor:(SFDS_COLOR_ALS)lmColor{
 [UIColor colorWithRed:0 green:0 blue:0 alpha:1],
 [UIColor colorWithRed:0 green:0 blue:0 alpha:0.05],
 [UIColor colorWithRed:0 green:0 blue:0 alpha:0.07],
+[UIColor colorWithRed:0 green:0 blue:0 alpha:0.1],
 [UIColor colorWithRed:0 green:0 blue:0 alpha:0.16],
+[UIColor colorWithRed:0 green:0 blue:0 alpha:0.2],
 [UIColor colorWithRed:0 green:0 blue:0 alpha:0.24],
 [UIColor colorWithRed:0 green:0 blue:0 alpha:0.4],
-[UIColor colorWithRed:1 green:1 blue:1 alpha:0.97],
-[UIColor colorWithRed:1 green:1 blue:1 alpha:0.75],
-[UIColor colorWithRed:1 green:1 blue:1 alpha:0.5],
+[UIColor colorWithRed:1 green:1 blue:1 alpha:0.1],
 [UIColor colorWithRed:1 green:1 blue:1 alpha:0.15],
+[UIColor colorWithRed:1 green:1 blue:1 alpha:0.2],
+[UIColor colorWithRed:1 green:1 blue:1 alpha:0.4],
+[UIColor colorWithRed:1 green:1 blue:1 alpha:0.5],
+[UIColor colorWithRed:1 green:1 blue:1 alpha:0.75],
+[UIColor colorWithRed:1 green:1 blue:1 alpha:0.97],
 [UIColor colorWithRed:1 green:1 blue:1 alpha:1],
+[UIColor colorWithRed:NaN green:NaN blue:NaN alpha:1],
```

Comment:
Everything looks good except one issue here. I will merge and fix it with a separate PR